server/src/index.ts

```diff
@@ -234,13 +235,27 @@ const apolloServer = new ApolloServer({
     DateTime: DateTimeResolver,
   },
   subscriptions: {
-    onConnect: (context: SubscriptionContext) => {
+    onConnect: async (context: SubscriptionContext) => {
       if (!context.token) {
         throw new Error('Missing auth token');
       }
       try {
-        jsonwebtoken.verify(context.token, config.jwtSecret);
-        // TODO ARTURS : FIND USER FN FOR EXTRA LAYER OF SECURITY
+        const decoded = jsonwebtoken.verify(
+          context.token,
+          config.jwtSecret
+        ) as {
+          userId: string;
+        };
+        const userId = decoded.userId;
+        try {
+          await prisma.user.findOne({
```

Comment:
@Akirtovskis I think the function will return null if the user is not found instead of throwing no?

Reply:
@pradel you are right, fixed.